docs/api/blockchain/introduction.mdx

```diff
@@ -53,19 +52,26 @@ next page:
 http get $(API_URL)/v1/blocks?cursor=eyJiZWZvcmUiOjQzOTIwMX0
 ```
 
-Which will return the next page of results and a next `cursor` for the following page. A cursor will
-be returned with each response until the last page is reached, at which point no cursor is returned.
+Which will return the next page of results and the next `cursor` for the following page. A cursor
+will be returned with each response until the last page is reached, at which point no cursor is
+returned.
 
 :::info
 
-In order to improve caching the API may return an empty data array and a cursor. Use the cursor to
-request more data.
+To improve caching the API may return an empty data array and a cursor. Use the cursor to request
+more data.
 
 :::
 
 ## Transaction Units
 
-All API endpoints which return a transaction value (i.e. `fee`, `amount`, etc) are represented in
-the base unit `bones`.
+Unless otherwise specified, API calls return values in the following units:
 
-The conversion rate is `100,000,000 bone` per `HNT`.
+| value        | Units        | Conversion to HNT             |
+| ------------ | ------------ | ----------------------------- |
+| `amount`     | bones        | `100,000,000 bone` to `1 HNT` |
+| `balance`    | bones        | `100,000,000 bone` to `1 HNT` |
+| `stake`      | bones        | `100,000,000 bone` to `1 HNT` |
+| `sum`        | bones        | `100,000,000 bone` to `1 HNT` |
+| `fee`        | Data Credits | `100,000 DC` to `1 HNT`       |
```

Comment:
this doesn't seem right. The price of Data Credits is fixed in USD (1 Data Credit = $0.00001). so 100,000 DC = $1 USD